src/lib/commands.js

```diff
@@ -54,11 +55,11 @@ exports.list = () => {
 
   files.forEach(file => {
     // eslint-disable-next-line global-require, import/no-dynamic-require
-    const command = require(file);
+    const Command = require(file);
 
     commands.push({
       file,
-      command,
+      command: new Command(),
```

Comment:
Thoughts on `new` here and load vs. `module.exports = new (commandClass)();` in each file?

I've used both techniques in the past. `new` here and load is nice to ensure that we don't have side effects leaking between calls, and might be what I prefer, but I figured I'd bring it up if you thought it would be useful.

Reply:
I think I prefer to do the instantiation on load instead of in the export.